include/NovelRTInterop/Maths/GeoVector4F_t.h

```diff
@@ -0,0 +1,296 @@
+    NovelRT::Maths::GeoVector4<float>v1 = NovelRT::Maths::GeoVector4<float>(vector.x, vector.y, vector.z, vector.w);
+    NovelRT::Maths::GeoVector4<float>v2 = NovelRT::Maths::GeoVector4<float>(other.x, other.y, other.z, other.w);
+    NovelRT::Maths::GeoVector4<float>v3 = NovelRT::Maths::GeoVector4<float>(epsilonValue.x, epsilonValue.y, epsilonValue.z, epsilonValue.w);
+    return v1.epsilonEquals(v2, v3);
+  }
+
+  inline GeoVector4F_t GeoVector4F_getNormalised(GeoVector4F_t& gv) {
+    NovelRT::Maths::GeoVector4<float> geo = NovelRT::Maths::GeoVector4<float>(gv.x, gv.y, gv.z, gv.w).getNormalised();
+    return { geo.getX(), geo.getY(), geo.getZ(), geo.getW() };
+  }
+
+  inline float GeoVector4F_getLength(GeoVector4F_t& gv) {
+    NovelRT::Maths::GeoVector4<float> geo = NovelRT::Maths::GeoVector4<float>(gv.x, gv.y, gv.z, gv.w);
+    return geo.getLength();
+  }
+
+  inline float GeoVector4F_getMagnitude(GeoVector4F_t& gv) {
+    return GeoVector4F_getLength(gv);
+  }
+
+  inline bool GeoVector4F_equal(const GeoVector4F_t& first, const GeoVector4F_t& other) {
+    return NovelRT::Maths::GeoVector4<float>(first.x, first.y, first.z, first.w) == NovelRT::Maths::GeoVector4<float>(other.x, other.y, other.z, other.w);
+  }
+
+  inline bool GeoVector4F_notEqual(const GeoVector4F_t& first, const GeoVector4F_t& other) {
+    return NovelRT::Maths::GeoVector4<float>(first.x, first.y, first.z, first.w) != NovelRT::Maths::GeoVector4<float>(other.x, other.y, other.z, other.w);
+  }
+
+  inline bool GeoVector4F_lessThan(const GeoVector4F_t& first, const GeoVector4F_t& other) {
+    return NovelRT::Maths::GeoVector4<float>(first.x, first.y, first.z, first.w) < NovelRT::Maths::GeoVector4<float>(other.x, other.y, other.z, other.w);
+  }
+
+  inline bool GeoVector4F_lessThanOrEqualTo(const GeoVector4F_t& first, const GeoVector4F_t& other) {
+    return NovelRT::Maths::GeoVector4<float>(first.x, first.y, first.z, first.w) <= NovelRT::Maths::GeoVector4<float>(other.x, other.y, other.z, other.w);
+  }
+
+  inline bool GeoVector4F_greaterThan(const GeoVector4F_t& first, const GeoVector4F_t& other) {
+    return NovelRT::Maths::GeoVector4<float>(first.x, first.y, first.z, first.w) > NovelRT::Maths::GeoVector4<float>(other.x, other.y, other.z, other.w);
+  }
+
+  inline bool GeoVector4F_greaterThanOrEqualTo(const GeoVector4F_t& first, const GeoVector4F_t& other) {
+    return NovelRT::Maths::GeoVector4<float>(first.x, first.y, first.z, first.w) >= NovelRT::Maths::GeoVector4<float>(other.x, other.y, other.z, other.w);
+  }
+
+  inline GeoVector4F_t GeoVector4F_addVector(GeoVector4F_t& first, const GeoVector4F_t& other) {
+    NovelRT::Maths::GeoVector4<float> geo = NovelRT::Maths::GeoVector4<float>(first.x, first.y, first.z, first.w)
+      + NovelRT::Maths::GeoVector4<float>(other.x, other.y, other.z, other.w);
+    first.x = geo.getX();
+    first.y = geo.getY();
+    first.z = geo.getZ();
+    first.w = geo.getW();
+    return first;
+  }
```

Comment:
There seems to be a lot of this as I previously confused this to be the same as `+=` - this is to be fixed.